rra2json.inc.json

```diff
@@ -15,7 +15,9 @@
 		"url": "https://bugzilla-dev.allizom.org/rest/",
 		"product": "",
 		"component": "",
-		"creator": ""
+		"creator": "",
+		"autoassign": [],
+		"cache": "/var/run/assignees.pickle"
```

Comment:
yeah, these are the ones mentioned in https://github.com/gdestuynder/rra2json/pull/5#issuecomment-298800194 

- you can keep the default for cache (or use something else if that path is not right)
- autoassign is a list of emails ["ameihm@mozilla.com", "tweir@mozilla.com", "jclaudius@mozilla.com", "gdestuynder@mozilla.com"]